src/app/public/modules/summary-action-bar/summary-action-bar.component.ts

```diff
@@ -177,12 +177,12 @@ export class SkySummaryActionBarComponent implements AfterViewInit, OnDestroy {
       });
     }
 
-    const config = { attribute: true, attributeFilter: ['hidden'], childList: false, characterDate: false };
+    const config = { attributes: true, attributeFilter: ['hidden'], childList: false, characterDate: false };
     let el = this.elementRef.nativeElement;
     do {
       if (el.classList.contains('sky-tab')) {
-        this.observer.observe(el, config);
-      }
+         this.observer.observe(el, config);
+       }
```

Comment:
Why are these spaced over?

Reply:
Honestly not sure. May have been from some commenting I removed. I've fixed it.